addons/website_slides/controllers/mail.py

```diff
@@ -77,3 +77,13 @@ def mail_update_message(self, thread_model, thread_id, message_id, post_data, **
             'default_attachment_ids': message.attachment_ids.sudo().read(['id', 'name', 'mimetype', 'file_size', 'access_token']),
             'force_submit_url': '/slides/mail/update_comment',
         }
+
+
+class SlidesMailController(MailController):
+
+    def _get_public_chatter_url_data(self, message):
+        if message.model == "slide.channel":
+            return {"url": f"/slides/{message.res_id}", "url_params": {"active_tab": "review"}}
+        if message.model == "slide.slide":
+            return {"url": f"/slides/slide/{message.res_id}"}
+        return super()._get_public_chatter_url_data(message)
```

Comment:
The pattern is usually not to override with a check on model, but rather to have the method in model/thread/portal mixin (depending on scope) and override it on each necessary model. This way we guarantee one override cannot impact another.

Isn't this pretty much a repeat of what is inside `_redirect_to_record` and by extension in `_get_access_action`?